utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala

```diff
@@ -44,9 +44,9 @@ object Main extends IOApp {
 
   val client = new GuardianContentClient(sys.env("CAPI_TEST_KEY"))
 
-  val assetsService: HttpRoutes[IO] = HttpRoutes.of[IO] {
+  def assetsService(blocker: Blocker): HttpRoutes[IO] = HttpRoutes.of[IO] {
```

Comment:
Again, how is this related to storyquestions? I'll have to look up what Blocker is and what it does.

Reply:
Ok, so I assume the `global` execution context was made available by `import org.http4s.implicits._`, and http4s `v0.21.0-M1` introduced a breaking change where `ExecutionContext` was replaced with "cats-effect-2.0’s new `Blocker`". Nothing at all obvious about any of that.

Reply:
yeah , this is def not clear , and i'm kind of sick of the cats-effect project at this stage. when i've got time i'll replace that with something a bit more usable.